exams/utils_test.py

```diff
@@ -23,43 +27,49 @@
 from profiles.factories import ProfileFactory
 
 
+def create_order(user, course_run):
+    """"
+    create payment for course
+    """
+    # pylint: disable=no-self-use
+    order = OrderFactory.create(user=user, status='fulfilled')
+    LineFactory.create(order=order, course_key=course_run.edx_course_key)
+    return order
+
+
 class ExamUtilTests(TestCase):
     """Tests for exam util"""
     @classmethod
     def setUpTestData(cls):
         with mute_signals(post_save):
             profile = ProfileFactory.create()
 
-        cls.program, cls.tier_programs = create_program(past=True)
+        cls.program, cls.tier_programs = create_program(
+            past=True,
+            exam_series_code="TestExam",
+            exam_module="SampleCourse"
+        )
         cls.user = profile.user
         cls.course_run = course_run = cls.program.course_set.first().courserun_set.first()
-        cls.enrollment = CachedEnrollmentFactory.create(user=cls.user, course_run=course_run)
-        CachedCurrentGradeFactory.create(
-            user=cls.user,
-            course_run=course_run,
-            data={
-                "passed": True,
-                "percent": 0.9,
-                "course_key": course_run.edx_course_key,
-                "username": cls.user.username
-            }
-        )
+        CachedEnrollmentFactory.create(user=cls.user, course_run=course_run)
+        with mute_signals(post_save):
```

Comment:
sorry I do not understand: what's the signal handler hooked to `CachedCurrentGrade`>